main.nf

```diff
@@ -722,18 +723,46 @@ process rseqc {
     """
     samtools index $bam_rseqc
     infer_experiment.py -i $bam_rseqc -r $bed12 > ${bam_rseqc.baseName}.infer_experiment.txt
-    RPKM_saturation.py -i $bam_rseqc -r $bed12  $strandRule -o ${bam_rseqc.baseName}.RPKM_saturation
     junction_annotation.py -i $bam_rseqc -o ${bam_rseqc.baseName}.rseqc -r $bed12
     bam_stat.py -i $bam_rseqc 2> ${bam_rseqc.baseName}.bam_stat.txt
     junction_saturation.py -i $bam_rseqc -o ${bam_rseqc.baseName}.rseqc -r $bed12 2> ${bam_rseqc.baseName}.junction_annotation_log.txt
     inner_distance.py -i $bam_rseqc -o ${bam_rseqc.baseName}.rseqc -r $bed12
-    geneBody_coverage.py -i $bam_rseqc -o ${bam_rseqc.baseName}.rseqc -r $bed12
     read_distribution.py -i $bam_rseqc -r $bed12 > ${bam_rseqc.baseName}.read_distribution.txt
     read_duplication.py -i $bam_rseqc -o ${bam_rseqc.baseName}.read_duplication
     echo "Filename $bam_rseqc RseQC version: "\$(read_duplication.py --version)
     """
 }
 
+/*
+ * Step 4.1 Rseqc genebody_coverage
+ */
+process genebody_coverage {
+    tag "${bam_geneBodyCoverage.baseName - '.sorted'}" 
+       publishDir "${params.outdir}/rseqc" , mode: 'copy',
+        saveAs: {filename ->
+                 if (filename.indexOf("bam_stat.txt") > 0)                      "bam_stat/$filename"
```

Comment:
Are these files not from the `bam_stat.py` process?

Reply:
You are correct